### ISSUE TITLE
Show ping indicator optional in nameplates of other tees

### ISSUE THREAD

**Issue by heinrich5991** [ddnet/ddnet#5355]
converted from #4835.

@Dumakulem wrote:
> add an option to show the ping before the name of the tee like cl_strong,
>
> the hud can look like a ball that has a color that represents on which ping the player has:
>
> 1-89 = green
> 90-149 = yellow
> 150-199 = orange
> 200+ = red
> timeout = gray circle
>
> the timeout one will be really useful to see if someone is currently on time out
>
> ![](https://user-images.githubusercontent.com/97694529/158855874-1045d52f-263d-439c-9fb5-3c58c9f6f542.png)
> this is how it looks like.. sorry i took a random photo then edited it on ms paint

@def- wrote:
> I'm against this, already too overloaded. Maybe we could show the name in red if the player had packet loss or increased ping in the last X seconds.

@sjrc6 wrote:
> This feature would be nice for gores where you need to save people differently depending on how much ping they have.

**Comment by C0D3D3V** [comment]
I would agree our nameplates are already very full. But a ping indicator would probably be more useful than cl_nameplates_ha.

We currently use the name colour for team colours. So if we would want to use name colours for ping indicator, ... that would disable the team colour display.

**Comment by kaitlynia** [comment]
I think the original idea OP had but reducing the size of the colored bubble would be a good solution. It doesn't compromise team colors and if people still find it to be cluttered, it's likely going to be a non-default anyway

**Comment by Robyt3** [comment]
Ping seems unnecessary to me. The connectivity could be shown though, see #9169.

**Comment by SollyBunny** [comment]
this is in tclient!

**Comment by SollyBunny** [comment]
> Ping seems unnecessary to me. The connectivity could be shown though, see [#9169](https://github.com/ddnet/ddnet/issues/9169).
@Robyt3 

I made the [opposite suggestion](https://github.com/ddnet/ddnet/pull/9172#discussion_r2103492160)
